pkg/fleet/internal/cdn/cdn.go

```diff
@@ -163,17 +182,3 @@ func (c *CDN) getOrderedLayers(ctx context.Context) ([]*layer, error) {
 
 	return orderedLayers, nil
 }
-
-func getHostTags(ctx context.Context, config model.Config) func() []string {
-	return func() []string {
-		// Host tags are cached on host, but we add a timeout to avoid blocking the RC request
-		// if the host tags are not available yet and need to be fetched. They will be fetched
-		// by the first agent metadata V5 payload.
-		ctx, cc := context.WithTimeout(ctx, time.Second)
-		defer cc()
-		hostTags := hosttags.Get(ctx, true, config)
-		tags := append(hostTags.System, hostTags.GoogleCloudPlatform...)
-		tags = append(tags, "installer:true")
-		return tags
-	}
-}
```

Comment:
Removed to better come back later when we do our own predicates 🚀

Reply:
Ah, would you prefer I leave this unused then? I assume the implementation with the predicates in the config body will be different enough, but I'm happy to leave it if you'd prefer

Reply:
Let's remove dead code, we can come back to this PR if we need that function back.